### PR TITLE
graphs/kruskal: add a test case to verify the correctness, fix styles

### PR DESCRIPTION
**Pull request by meysam81**
And also to increase the coverage as stated in #2128 

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: graphs/minimum_spanning_tree_kruskal.py [hunk 1]
```diff
@@ -20,13 +12,22 @@ def find_parent(i):
...
-        print(edge)
+    return minimum_spanning_tree
+
+
+if __name__ == "__main__":  # pragma: no cover
+    num_nodes, num_edges = list(map(int, input().strip().split()))
+    edges = []
+
+    for _ in range(num_edges):
+        node1, node2, cost = list(map(int, input().strip().split()))
```
Comment (cclauss) [inline graphs/minimum_spanning_tree_kruskal.py:30]:
```suggestion
        node1, node2, cost = [int(x) for x in input().strip().split()]
```
As discussed in CONTRIBUTING.md

File: graphs/tests/test_min_spanning_tree_kruskal.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+             [7, 6, 1],
+             [2, 8, 2],
+             [8, 6, 6],
+             [2, 3, 7],
+             [2, 5, 4],
+             [6, 5, 2],
+             [3, 5, 14],
+             [3, 4, 9],
+             [5, 4, 10],
+             [1, 7, 11]]
```
Comment (cclauss) [inline graphs/tests/test_min_spanning_tree_kruskal.py:19]: psf/black will reformat the matrices.

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Comment by cclauss** [comment]
No type hints or doctests.

**Comment by poyea** [comment]
Right we can add some type hints to this file @meysam81

File: graphs/minimum_spanning_tree_kruskal.py [hunk 3]
```diff
@@ -1,13 +1,5 @@
...
-    num_nodes, num_edges = list(map(int, input().strip().split()))
-
-    edges = []
-
-    for i in range(num_edges):
-        node1, node2, cost = list(map(int, input().strip().split()))
-        edges.append((i, node1, node2, cost))
-
-    edges = sorted(edges, key=lambda edge: edge[3])
+def run_algorithm(num_nodes, num_edges, edges):
```
[the remark this answers is not in the document]
Comment (meysam81) [reply graphs/minimum_spanning_tree_kruskal.py]: Adding type hints was done in #3101 which you merged a couple of hours ago.
We can remove `num_edges` if you say so.

[the remark this answers is not in the document]
Comment (cclauss) [reply graphs/minimum_spanning_tree_kruskal.py]: These comments were on my second computer and I never hit Send...  Thanks for doing this.

[the remark this answers is not in the document]
Comment (meysam81) [reply graphs/minimum_spanning_tree_kruskal.py]: My pleasure :clinking_glasses:

**Pull request merged**